gym/lib/gym/runner.rb

```diff
@@ -271,8 +271,7 @@ def copy_files_from_path(path)
 
     # Copies the .app from the archive into the output directory
     def copy_mac_app
-      exe_name = Gym.project.build_settings(key: "EXECUTABLE_NAME")
-      app_path = File.join(BuildCommandGenerator.archive_path, "Products/Applications/#{exe_name}.app")
+      app_path = Dir[File.join(BuildCommandGenerator.archive_path, "Products/Applications/*.app")].last
```

Comment:
Do you know if there is any chance that there could be multiple `.app`s that are created or there could be a previous version that existed with a different name that this could be picking up? 🤔 

Maybe this needs to pick up the newest `.app` file from the directory? 🤷‍♂️  Thoughts?

Reply:
I'll do some research on how Xcode handles multiple apps in the same workspace/project to see if it is possible to have more than one.

Another option is that Fastlane could fallback to the last `.app` found only if an app cannot be found with the `EXECUTABLE_NAME`.

Reply:
Ohhhh, I would feel a lot better if we used the last `.app` as a fallback 😇 Do you mind changing it up to do that?

Reply:
@joshdholtz This is ready for another look when you get a chance.

Reply:
@joshdholtz Don't want to be a bother but bumping this in case you missed the last message.